REM-1296: Get list of the batches (#12)
Remmeauth/remme-core-cli: README.md

    * [Nodes](#nodes)
    * [Configuration file](#configuration-file)
    * [Service](#service)

>     * [Batch](#batch)

Reviewer: Why account has been removed?

Author: Done.

368440.0
```

> ### Blockchain info

Reviewer: When you operate some libraries where you see `blockchain info`, it is ok.
When you provide the independent layer of user experience, you should care about users in own way.

I mean, In the library it is blockchain information, but CLI user expects section batch.

Author: Done.

368440.0
```

> ### Batches

Reviewer 2: Add anchor `Batches` to the tree.

Author: Done.

> Get batches — ``remme batch get-list``:

Reviewer 2: ```suggestion
Get list of batches — ``remme batch get-list``:
```

Author: Done.

| Arguments | Type   |  Required | Description                                         |
| :-------: | :----: | :-------: | --------------------------------------------------- |

> | id        | String |  No       | Get batch by its identifier.                                |

Reviewer 2: ```suggestion
| id        | String |  No       | Identifier to get a list of batches by.                                |
```
Check all description by this issue.

Author: Done.

}
```

### Batch

Get a list of batches — ``remme batch get-list``:

| Arguments | Type   |  Required | Description                                         |
| :-------: | :----: | :-------: | --------------------------------------------------- |
| ids       | String |  No       | Identifiers to get a list of batches by.            |
| start     | String |  No       | Parameter to list batches starting from.            |

> | limit     | Integer|  No       | Parameter to limit amount of batches.               |

Reviewer: Is `start` an identifier of the `block` to start paging (inclusive) from?

`Parameter to limit amount of batches` > `Number of transactions to return`.

Author: 1) start - Batch identifier to start listing from.
2) Maximum amount of batches to return.

| head      | String |  No       | Block identifier to get a list of batches from.     |
| reverse   | String |  No       | Parameter to reverse result.                        |
| node-url  | String |  No       | Node URL to apply the command to.                   |

```bash
$ remme batch get-list \
      --ids=[c2eeb94926d3432e41cb5ceed078f78466389e4fe685ec958021a1368f634c035072e434e5d0ff64820dd01fde6e5afc67eb5a9ae6c48d3983ff43abe98aef6b] \

>       --node-url=159.89.104.9

Reviewer: Provide the most full example. As [here](https://github.com/Remmeauth/remme-core-cli/pull/18/files#diff-04c6e90faac2675aa89e2176d2eec7d8R137).

Author: Done.

| :-------: | :----: | :-------: | ------------------------------------------------------|
| ids       | String |  No       | Identifiers to get a list of batches by.              |
| start     | String |  No       | Batch identifier to get a list batches starting from. |
| limit     | Integer|  No       | Maximum amount of batches to return.                  |
| head      | String |  No       | Block identifier to get a list of batches from.       |
| reverse   | Bool   |  No       | Parameter to reverse result.                          |
| node-url  | String |  No       | Node URL to apply the command to.                     |

```bash
$ remme batch get-list \
      --ids=[c2eeb94926d3432e41cb5ceed078f78466389e4fe685ec958021a1368f634c035072e434e5d0ff64820dd01fde6e5afc67eb5a9ae6c48d3983ff43abe98aef6b] \
      --start=c2eeb94926d3432e41cb5ceed078f78466389e4fe685ec958021a1368f634c035072e434e5d0ff64820dd01fde6e5afc67eb5a9ae6c48d3983ff43abe98aef6b \
      --limit=1 \
      --head=56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551 \
      --reverse=true \

>       --node-url=159.89.104.9

Reviewer: Use common URL as in other examples — genesis.

| ids       | String |  No       | Identifiers to get a list of batches by.              |
| start     | String |  No       | Batch identifier to get a list batches starting from. |
| limit     | Integer|  No       | Maximum amount of batches to return.                  |
| head      | String |  No       | Block identifier to get a list of batches from.       |
| reverse   | Bool   |  No       | Parameter to reverse result.                          |
| node-url  | String |  No       | Node URL to apply the command to.                     |

```bash
$ remme batch get-list \
      --ids='6bd3382e3deef34d0bc63a7b450c88c7ae00152f5168c7b4dc4357feff6d52175209919cd0710441fa2768f4c12adf97143440ef8414bb5144b9459d78ff3e0e, 7a5daba99d5757adc997ea6a0b1b83263b3c16604dbd83c0153dc01c9fd780af4b570338c2ec60e086b1db58a4397a4dc661d6c93b0a7250fe75642e15b26e81' \
      --start=6bd3382e3deef34d0bc63a7b450c88c7ae00152f5168c7b4dc4357feff6d52175209919cd0710441fa2768f4c12adf97143440ef8414bb5144b9459d78ff3e0e \
      --limit=2 \
      --head=57a7944497ca41f424932ae6b70897e7086652ab98450d4aba6a02a2d891501460947812a41028b8041f087066df6dc7e1100c4b0e5cc94bb58b002f6950eb02 \

>       --node-url=node-genesis-testnet.remme.io \

Reviewer 2: ```suggestion
      --reverse \
      --node-url=node-genesis-testnet.remme.io 
```

| start       | String |  No       | Batch identifier to get a list of batches starting from.|
| limit       | Integer|  No       | Maximum amount of batches to return.                    |
| head        | String |  No       | Block identifier to get a list of batches from.         |
| reverse     | Bool   |  No       | Parameter to reverse result.                            |
| node-url    | String |  No       | Node URL to apply a command to.                         |

```bash
$ remme batch get-list \
      --ids='6bd3382e3deef34d0bc63a7b450c88c7ae00152f5168c7b4dc4357feff6d52175209919cd0710441fa2768f4c12adf97143440ef8414bb5144b9459d78ff3e0e, 7a5daba99d5757adc997ea6a0b1b83263b3c16604dbd83c0153dc01c9fd780af4b570338c2ec60e086b1db58a4397a4dc661d6c93b0a7250fe75642e15b26e81' \
      --start=6bd3382e3deef34d0bc63a7b450c88c7ae00152f5168c7b4dc4357feff6d52175209919cd0710441fa2768f4c12adf97143440ef8414bb5144b9459d78ff3e0e \
      --limit=2 \
      --head=57a7944497ca41f424932ae6b70897e7086652ab98450d4aba6a02a2d891501460947812a41028b8041f087066df6dc7e1100c4b0e5cc94bb58b002f6950eb02 \
      --node-url=node-6-testnet.remme.io \
      --reverse

>  "result": {

Reviewer 2: Why is the result not in the dictionary?
```bash
{
    'result': ...
}
```

Author: Done.

                            "00b10c0100000000000000000000000000000000000000000000000000000000000000",
                            "00b10c00"
                        ],
                        "payload_sha512": "7b11153de66545d8c8847004425f9c5815483636688e79fd2bfbb6d979218fbeb7ccdcb244241d8d52ea38a1b1d62c5d178cf74c3c7b5f496936059c616163e2",
                        "signer_public_key": "03738df3f4ac3621ba8e89413d3ff4ad036c3a0a4dbb164b695885aab6aab614ad"
                    },
                    "header_signature": "376efc69c217a0b9deb545348ca32664ce61b3e35706252d1d0374bdb93b10e62abc35fc16a3d19f0d8346ddbadc1c0974af6b4364f98ffea66de72cfb11b238",
                    "payload": "CtMCCLwbEoABYmZiNzhkNGQxMWQyZjQzOWRlZjkzNTc2Y2YyN2M0NGVhZTNmYmMyM2Q2ODAwNmUyNGRlYmJmZGYxZWRiNmQ4MDY5MDExYzYxNWZjNjk4NGMxM2EzZDJjMDMyYzFhZTY2NWYzNmZjZTUxOWVjZTdlOGI2YmFjMGMxYWRlMTgxYWYaQjAzNzM4ZGYzZjRhYzM2MjFiYThlODk0MTNkM2ZmNGFkMDM2YzNhMGE0ZGJiMTY0YjY5NTg4NWFhYjZhYWI2MTRhZCKAAThmMjJkYjUyNTUyYzQ3MjE4ZTc0ZmE4OGExZTU2NGJhZTE1YjYwMmY0ZTI3ZTZiYTYwOWI0NzM4YjY0ZTllZTYzYzcwMjM4MjI3ZWU0NTU1OTVhNjMzYTIzOWU5ZGZiMWNiMGMxNWI1MzVhZGJkYTZmMGE2Yjk3MmU3ZWU3MWQyKLvr4eYF"
                }
            ]
        }
    ]
}

> 

Reviewer 2: Remove the blank line.

Author: Done.

| ids         | String |  No       | Identifiers to get a list of batches by.                |
| start       | String |  No       | Batch identifier to get a list of batches starting from.|
| limit       | Integer|  No       | Maximum amount of batches to return.                    |
| head        | String |  No       | Block identifier to get a list of batches from.         |
| reverse     | Bool   |  No       | Parameter to reverse result.                            |
| node-url    | String |  No       | Node URL to apply a command to.                         |

```bash
$ remme batch get-list \
      --ids='6bd3382e3deef34d0bc63a7b450c88c7ae00152f5168c7b4dc4357feff6d52175209919cd0710441fa2768f4c12adf97143440ef8414bb5144b9459d78ff3e0e, 7a5daba99d5757adc997ea6a0b1b83263b3c16604dbd83c0153dc01c9fd780af4b570338c2ec60e086b1db58a4397a4dc661d6c93b0a7250fe75642e15b26e81' \
      --start=6bd3382e3deef34d0bc63a7b450c88c7ae00152f5168c7b4dc4357feff6d52175209919cd0710441fa2768f4c12adf97143440ef8414bb5144b9459d78ff3e0e \
      --limit=2 \
      --head=57a7944497ca41f424932ae6b70897e7086652ab98450d4aba6a02a2d891501460947812a41028b8041f087066df6dc7e1100c4b0e5cc94bb58b002f6950eb02 \

>       --node-url=node-6-testnet.remme.io \

Reviewer 2: ```suggestion
      --reverse \
      --node-url=node-6-testnet.remme.io
```
Node URL must be last.

Author: Done.
